### PR TITLE
Harmony: fix HistoryRecord

### LINKED ISSUE #1017: Harmony: fix HistoryRecord
1) in strict mode the height is set to 0
2) incorrect positioning of the avatar relative to the divider

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Bug Fix

## Related issues

Resolve #1017 

Related issues:
- https://github.com/taskany-inc/issues/issues/2562

### CODE REVIEW

File: src/harmony/HistoryRecord/HistoryRecord.module.css [hunk 1]
```diff
@@ -97,4 +94,14 @@
...
 .HistoryRecordCollapse + .HistoryRecord {
     align-items: center;
-}
\ No newline at end of file
+}
+
+.HistoryRecordCollapse {
+    height: 0;
+    overflow: hidden;
+    transition: height var(--transition-func) var(--transition-time);
```
Comment (tsumo) [inline src/harmony/HistoryRecord/HistoryRecord.module.css:102]: I thought `transition: height` to `auto` doesn't work in css

Comment (DenisVorop) [reply src/harmony/HistoryRecord/HistoryRecord.module.css:101]: yeah
height in css file are redundant styles
i removed it now

**Review by tsumo — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**